tutorials/python/python-type-driven-development/tutorial.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Type Driven Development in Python
+published: 2019-08-01
+tags: Python, typing, types, tdd
+language: python
+author-name: Carlos Villavicencio
+twitter-profile: po5i
+github-profile: po5i
+description: Learn how to improve the software quality and readability by using type driven development in Python.
```

Comment:
I think that this description could be better if you add more details. Also, after reading the tutorial, are the examples actually about doing type-driven development or more like how to use types in Python? If the idea is to make it more about type-driven development, perhaps a bigger example could be used to illustrate how to do that.

Reply:
You're right. The goal of this tutorial is to provide examples and encourage the reader on how to use types in Python instead of going deep in type-driven development. I changed a bit the introductory paragraphs to do so to avoid confusion. Thanks for your review. \cc @jsl